src/backend/distributed/planner/fast_path_router_planner.c

```diff
@@ -61,6 +63,19 @@ static bool ConjunctionContainsColumnFilter(Node *node,
 											Var *column,
 											Node **distributionKeyValue);
 
+void
```

Comment:
just to remind the point we discussed - we can consider adding a function comment for this:

```c
/*
 * FastPathPreprocessParseTree ..
 */
FastPathPreprocessParseTree(Query *parse)
```